.claude/statusline.sh

```
@@ -0,0 +1,141 @@
+  elif [ "$n" -ge 1000 ] 2>/dev/null; then
+    awk "BEGIN {printf \"%.1fk\", $n/1000}"
+  else
+    echo "${n:-0}"
+  fi
+}
+
+# Initialize usage log
+[ ! -f "$USAGE_LOG" ] && echo "ts,sid,tokens" > "$USAGE_LOG"
+
+# Session & burn rate tracking
+br_val=0
```

Shellcheck (0.11.0)

[warning] br_val appears unused. Verify use (or export if used externally).

(SC2034)